lib/src/components/NxSubmitMask/__tests__/NxSubmitMask.test.tsx

```diff
@@ -11,14 +11,13 @@ import NxLoadingSpinner from '../../NxLoadingSpinner/NxLoadingSpinner';
 import NxSubmitMask from '../NxSubmitMask';
 
 describe('NxSubmitMask', function() {
-  it('renders a .nx-submit-mask div containing a .nx-submit-mask__message h3 containing a NxLoadingSpinner',
+  it('renders a .nx-submit-mask div containing a .nx-submit-mask__message div containing a NxLoadingSpinner',
       function() {
         const component = shallow(<NxSubmitMask/>);
 
         expect(component).toMatchSelector('div.nx-submit-mask');
-        expect(component.find('h3')).toHaveClassName('nx-h3');
-        expect(component.find('h3')).toHaveClassName('nx-submit-mask__message');
-        expect(component.find('h3').find(NxLoadingSpinner)).toExist();
+        expect(component.find('div.nx-submit-mask div')).toHaveClassName('nx-submit-mask__message');
+        expect(component.find('div.nx-submit-mask__message').find(NxLoadingSpinner)).toExist();
```

Comment:
Btw you don't need to add the `div` part at the beginning, you can start with just the classname `.nx-submit-mask__message`.  That's what we'd typically do

Reply:
> Btw you don't need to add the `div` part at the beginning, you can start with just the classname `.nx-submit-mask__message`. That's what we'd typically do

It didn't work for me before, but now it does so I must have screwed something else up: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/7eaad8766f8001a37f8fff6f890d10ea7a8be8af